GliaWidgets/Sources/Interactor/Interactor.swift

```diff
@@ -70,6 +70,11 @@ class Interactor {
     }
     var environment: Environment
 
+    /// Skips Live Observation confirmation alerts/snackbars in
+    /// specific cases. For example after restarting engagement
+    /// after authentication.
+    var skipLOConfirmations = false
```

Comment:
I would prefer if this is called `skipLiveObservationConfirmations`. Imagine someone new comes and doesn't know what LO is, even worse that it seems like some marketers have decided to change the name to CoView or whatever

Reply:
updated ccc5b414f1c1ef4fcab1b1fe772e12e33a7db626